docs/Storyboards.md

```diff
@@ -4,7 +4,7 @@
 - Pick `Navigation` from the dropdown and set bundle to `com.mapbox.MapboxNavigation`.
 - Set up a segue to the storyboard reference like you would to any other UIViewController.
 
-<img src="https://cloud.githubusercontent.com/assets/764476/23622518/e3cc8a86-0253-11e7-80ab-7d34302a5fe5.png" width=340>
+<img src="https://cloud.githubusercontent.com/assets/764476/25007627/6467a1f0-2061-11e7-9349-cd25264c6ec6.png" width=340>
```

Comment:
Where does this image come from? If you got the URL by dragging an image into a comment or issue that you didn’t end up saving, GitHub could garbage-collect this image at any time. (This is also true of the screenshot in the readme.)

Instead, we should check in any screenshots we use in documentation and refer to them in Markdown documents using relative paths.

/cc @bsudekum